src/test/java/com/andreamazzarella/contact_manager/InMemoryRepositoryTest.java

```diff
@@ -0,0 +1,61 @@
+package com.andreamazzarella.contact_manager;
+
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static com.andreamazzarella.contact_manager.helpers.InMemoryRepositoryHelper.createContactInRepository;
+import static org.junit.Assert.assertEquals;
+
+
+public class InMemoryRepositoryTest {
+    private InMemoryRepository testRepo = new InMemoryRepository();;
+    private Contact andrea = createContactInRepository(testRepo, new String[] {"name", "Andrea"});
```

Comment:
Better to use types instead of a string array. They give you more verification that your program does the right thing.